### DISCUSSION TITLE
Create Team membership using phone number

### DISCUSSION THREAD

**Discussion by ursrudra** [appwrite/appwrite#5121] [Ideas]
Currently, the Teams API is using email to create membership and send the verification email. 
Are there any plans to add phone number support as well? So that we can use either a phone number or email address to create a team member.
This option is more useful when we use phone authentication and no need to collect extra email information from users.

**Comment by stnguyen90** [comment]
Please 👍🏼 [this issue](https://github.com/appwrite/appwrite/issues/3570) so that we can properly prioritize this need.

**Reply by ursrudra** [reply]
sure
